src/wp-admin/includes/plugin.php

```diff
@@ -1203,26 +1203,39 @@ function validate_plugin_requirements( $plugin ) {
 	}
 
 	if ( WP_Plugin_Dependencies::has_unmet_dependencies( $plugin ) ) {
-		$dependencies       = WP_Plugin_Dependencies::get_dependencies( $plugin );
-		$unmet_dependencies = array();
+		$dependency_names       = WP_Plugin_Dependencies::get_dependency_names( $plugin );
+		$unmet_dependencies     = array();
+		$unmet_dependency_names = array();
 
-		foreach ( $dependencies as $dependency ) {
+		foreach ( $dependency_names as $dependency => $dependency_name ) {
 			$dependency_file = WP_Plugin_Dependencies::get_dependency_filepath( $dependency );
 
 			if ( false === $dependency_file ) {
-				$unmet_dependencies['not_installed'][] = $dependency;
+				$unmet_dependencies['not_installed'][ $dependency ] = $dependency_name;
+				$unmet_dependency_names[]                           = $dependency_name;
 			} elseif ( is_plugin_inactive( $dependency_file ) ) {
-				$unmet_dependencies['inactive'][] = $dependency;
+				$unmet_dependencies['inactive'][ $dependency ] = $dependency_name;
+				$unmet_dependency_names[]                      = $dependency_name;
 			}
 		}
 
+		$plugins_page  = is_multisite() ? network_admin_url( 'plugins' ) : admin_url( 'plugins' );
+		$error_message = sprintf(
+			/* translators: 1: Plugin name, 2: Number of plugins, 3: A comma-separated list of plugin names, 4: Link to the plugins page. */
+			_n(
+				'<strong>Error:</strong> %1$s requires %2$d plugin to be installed and activated: %3$s. <a href="%4$s>"Manage plugins</a>',
+				'<strong>Error:</strong> %1$s requires %2$d plugins to be installed and activated: %3$s. <a href="%4$s>"Manage plugins</a>',
+				count( $unmet_dependency_names )
+			),
+			$plugin_headers['Name'],
+			count( $unmet_dependency_names )
```

Comment:
```suggestion
			count( $unmet_dependency_names ),
```

to fix the fatal error